README.adoc

```diff
@@ -0,0 +1,92 @@
+= vse-sync-pp
+
+This repository contains python modules for analyzing and generating synchronization test results by means of consuming input data coming from:
+
+* ptp4l, ts2phc, and phc2sys logs coming from PTP operator logs
+
+* collector files generated by collector tools developed in https://github.com/redhat-partner-solutions/vse-sync-collection-tools[collectors]
+
+These are the main python modules:
+
+* link:src/vse_sync_pp/demux.py[demux]: Demultiplex gathered data messages of a given identifier coming from a single multiplexed data source. Data source is a collector file. For each demultiplexed data message of a given identifier print the canonical data produced by the parser as JSON using `parse` module.
```

Comment:
Not sure I understand when should I use `demux` or `parse`

Reply:
Improved text a bit.